client/pages/Event/EventsPage.jsx

```diff
@@ -45,18 +46,55 @@ class EventsPage extends React.Component {
     });
   }
 
-  componentWillReceiveProps(nextProps) {
+  /**
+   *
+   *
+   * @param {*} prevProps
+   * @param {*} prevState
+   * @param {*} snapshot
+   * 
+   * @memberof EventsPage
+   */
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (this.btnRef.current !== null) {
+      if (snapshot !== null) {
+        const btn = this.btnRef.current;
+        btn.scrollIntoView();
+      }
+    }
+  }
+
+  /**
+   *
```

Comment:
Also here `https://github.com/AndelaOSP/Andela-Socials/pull/141/files#r230769859`
Mind adding some comment to document what this function does ?